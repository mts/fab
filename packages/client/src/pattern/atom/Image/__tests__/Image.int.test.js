import { defaultRender } from './Image.int.render'

describe('<Image />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
