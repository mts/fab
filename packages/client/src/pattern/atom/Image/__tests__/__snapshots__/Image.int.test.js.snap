// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> Snaphot must match catImageRender 1`] = `
<div
  className="image"
>
  <img
    alt="beng"
    src="https://66.media.tumblr.com/tumblr_m4iejdP8Hb1qd477zo1_250.jpg"
  />
</div>
`;

exports[`<Image /> Snaphot must match dogImageRender 1`] = `
<div
  className="image"
>
  <img
    alt="collie"
    src="https://images.dog.ceo/breeds/collie-border/n02106166_1031.jpg"
  />
</div>
`;
